web/cypress/integration/classifications/classifications.spec.js

```diff
@@ -1,151 +1,126 @@
 context('TASKANA Classifications', () => {
   beforeEach(() => cy.loginAs('admin'));
 
-  it('should be able to visit Classifications and filter by manual', () => {
+  it('should be possible to edit the Service Level of a classification', () => {
     cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
+    cy.wait(Cypress.env('pageReload'));
```

Comment:
I don't want to blow up the scope of this ticket. But Cypress states that this is a bad practice.
See: https://docs.cypress.io/guides/references/best-practices.html#Unnecessary-Waiting

We should follow those best practices and rework the wait command to use aliases instead of an arbitrary number.

Do you want to rework this in this ticket or create a follow up ticket?

Reply:
When using the cy.visit it is not possible to work with an alias. 
However, I have found an alternative solution:
cy.location('hash', {timeout: 10000}).should('include', '/page');

In addition, I can delete the explicit page reloads after saving since this does not correspond to a common user behavior.